src/RRule.php

```diff
@@ -2016,11 +2016,21 @@ static protected function i18nList(array $array, $and = 'and')
 	 * Will load the basic first (e.g. "en") and then the region-specific if any
 	 * (e.g. "en_GB"), merging as necessary.
 	 * So region-specific translation files don't need to redefine every strings.
+	 *
+	 * @param string $locale
+	 * @param string $fallbackLocale
+	 *
+	 * @return array
+	 * @throws \InvalidArgumentException
 	 */
-	static protected function i18nLoad($locale)
+	static protected function i18nLoad($locale, $fallbackLocale = 'en_US')
```

Comment:
I write the method names in camelCase, but the functions and variables in snake_case, so you should rename this $fallback_locale (or just $fallback is you don't like snake_case)